lua/bufferline/render.lua

```diff
@@ -278,7 +247,7 @@ function render.close_buffer_animated(bufnr)
   end
 
   local buffer_data = state.get_buffer_data(bufnr)
-  local current_width = buffer_data.real_width
+  local current_width = buffer_data.real_width or 0
```

Comment:
`real_width` has the potential to be `nil`, but `animate.start` does not account for a `nil` value. The `or 0` should never happen, but it will prevent an error in the case that it does.